tests/FSharp.Compiler.ComponentTests/Interop/DeeplyNestedCSharpClasses.fs

```diff
@@ -33,7 +33,7 @@ namespace MyNamespace
 
         let fsharpSource =
             """
-let loss2 = MyNamespace.OoterClass.InnerClass.MoreInnerClass.somefunction()   //Note the miss-typed functionname we expect a good error message
+let loss2 = MyNamespace.OoterClass.InnerClass.MoreInnerClass.somefunction()   //Note the mistyped functionname we expect a good error message
```

Comment:
I guess this was meant to be OuterClass

Reply:
No, here the comment is wrong.

```f#
let loss2 = MyNamespace.OoterClass.InnerClass.MoreInnerClass.somefunction()   //Note the miss-typed functionname we expect a good error message
"""
...
        |> withSingleDiagnostic (Error 39, Line 2, Col 25, Line 2, Col 35, "The value, constructor, namespace or type 'OoterClass' is not defined. Maybe you want one of the following:
   OuterClass")
```
So the line should be:
```suggestion
let loss2 = MyNamespace.OuterClass_.InnerClass.MoreInnerClass.somefunction()   //Note the mistyped class name we expect a good error message
```

Reply:
Yeah also didn't notice, probably can also keep then

Reply:
Well, I'm definitely going to fix the comment since that wasn't helpful